integration/testnet/util.go

```diff
@@ -105,3 +106,26 @@ func WriteJSON(path string, data interface{}) error {
 	err = ioutil.WriteFile(path, marshaled, 0644)
 	return err
 }
+
+// rootProtocolJsonWithoutAddresses strips out all node addresses from the root protocol json file specified as srcFile
+// and creates the dstFile with the modified contents
+func rootProtocolJsonWithoutAddresses(srcfile string, dstFile string) error {
+
+	fileData, err := ioutil.ReadFile(srcfile)
+	if err != nil {
+		return err
+	}
+	fileString := string(fileData)
+	// look for all "Address": "<node addres>" and remove those elements
+	m1 := regexp.MustCompile(`.*"Address".*:.*".*".*\n`)
+	newString := m1.ReplaceAllString(fileString, "")
+	newFileData := []byte(newString)
```

Comment:
I think parsing the JSON into a struct / map, removing the fields, and then rewriting the JSON to the file is cleaner.

Here you're relying on the fact that each JSON key is on it's own line, which is not necessarily true. It is valid to have multiple keys on the same line, and the regex here would fail in that case.